website/source/docs/guides/minikube.md

```diff
@@ -0,0 +1,259 @@
+
+In this guide, you'll start a local Kubernetes cluster with minikube. You'll install Consul with only a few commands, then deploy two custom services that use Consul to discover each other over encrypted TLS via Consul Connect. Finally, you'll tighten down Consul Connect so that only the approved applications can communicate with each other.
+
+[Demo code](https://github.com/hashicorp/demo-consul-101) is available.
+
+- [Task 1: Start Minikube and Install Consul with Helm](#task-1-start-minikube-and-install-consul-with-helm)
+- [Task 2: Deploy a Consul Aware Application to the Cluster](#task-2-deploy-a-consul-aware-application-to-the-cluster)
+- [Task 3: Configure Consul Connect](#task-3-use-consul-connect)
+
+## Prerequisites
+
+Let's install Consul on Kubernetes with minikube. This is a relatively quick and easy way to try out Consul on your local machine without the need for any cloud credentials. You'll be able to use most Consul features right away.
+
+First, you'll need to follow the directions for [installing minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/), including VirtualBox or similar.
+
+You'll also need to install `kubectl` and `helm`.
+
+Mac users can install `helm` and `kubectl` with Homebrew.
+
+```sh
+$ brew install kubernetes-cli
+$ brew install kubernetes-helm
+```
+
+Windows users can use Chocolatey with the same package names:
+
+```sh
+$ choco install kubernetes-cli
+$ choco install kubernetes-helm
+```
+
+For more on Helm, see [helm.sh](https://helm.sh/).
+
+## Task 1: Start Minikube and Install Consul with Helm
+
+### Step 1: Start Minikube
+
+Start minikube. You can use the `--memory` option with the equivalent of 4GB to 8GB so there is plenty of memory for all the pods we will run. This may take several minutes. It will download a 100-300MB of dependencies and container images.
+
+```
+$ minikube start --memory 4096
+```
+
+Next, let's view the local Kubernetes dashboard with `minikube dashboard`. Even if the previous step completed successfully, you may have to wait a minute or two for minikube to be available. If you see this message, try again.
+
+Once it spins up, you'll see the dashboard in your web browser. You can view pods, nodes, and other resources.
+
+```
+$ minikube dashboard
+```
+
+![Minikube Dashboard](/assets/images/guides/minikube-dashboard.png "Minikube Dashboard")
+
+### Step 2: Install the Consul Helm Chart to the Cluster
+
+To perform the steps in this lab exercise, clone the [hashicorp/demo-consul-101](https://github.com/hashicorp/demo-consul-101) repository from GitHub. Change into the repo, and go to the `k8s` directory inside.
```

Comment:
"Change into the repo..."

Is this missing a word?

Reply:
Should be "change into the cloned directory" or similar. Will fix.

Reply:
Fixed in a2d0b7c41